packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -2182,8 +2188,7 @@ describe.each([
         }).toContainExactly([{ name: "baz", productCat: undefined }])
       })
     })
-
-  isInternal &&
+  ;(isSqs || isLucene) &&
```

Comment:
Semi-colon is annoyingly required or Node parses this as an attempted function call.

Reply:
💩 yes, I really dislike this